README.md

```diff
@@ -1,2 +1,37 @@
-# ocean-acidification
-Landing page for IOOS Ocean Acidification
+# HF Radar DAC
```

Comment:
Please update the README so it refers to the Ocean Acidification project rather than HF Radar DAC